doc/usecases/geo_map.md

```diff
@@ -0,0 +1,26 @@
+# Use Case: GeoMap
+
+## Summary
+
+- **Scope:** GeoMap
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user can view sites/maps on a GeoMap
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  - The user has navigated to the map:
+  The user is either logged into the PermaPlanT Nextcloud instance and clicked on the "Maps" tab or
```

Comment:
I'm sure that this is probably not the case, but just to double-check: should maps be available only to logged-in users?

Reply:
Good point, some maps are visible without being logged in and the geo map (should be added in glossary, I think currently it is called Nextcloud map there) definitely should work without logged in, as it even is on the landing page.